test_scripts/MobileProjection/Phase2/common.lua

```diff
@@ -0,0 +1,141 @@
+---------------------------------------------------------------------------------------------------
+-- Common module
+---------------------------------------------------------------------------------------------------
+--[[ General configuration parameters ]]
+config.defaultProtocolVersion = 2
+
+--[[ Required Shared libraries ]]
+local actions = require("user_modules/sequences/actions")
+local utils = require("user_modules/utils")
+local test = require("user_modules/dummy_connecttest")
+local events = require("events")
+
+--[[ Module ]]
+local m = actions
+
+m.failedTCs = {}
+
+m.wait = utils.wait
+
+function m.setAppConfig(pAppId, pAppHMIType, pIsMedia)
+  m.getConfigAppParams(pAppId).appHMIType = { pAppHMIType }
+  m.getConfigAppParams(pAppId).isMediaApplication = pIsMedia
+end
+
+function m.registerApp(pAppId)
+  if not pAppId then pAppId = 1 end
+  m.getMobileSession(pAppId):StartService(7)
+  :Do(function()
+      local corId = m.getMobileSession(pAppId):SendRPC("RegisterAppInterface", m.getConfigAppParams(pAppId))
+      m.getHMIConnection():ExpectNotification("BasicCommunication.OnAppRegistered",
+        { application = { appName = m.getConfigAppParams(pAppId).appName } })
+      :Do(function(_, d)
+          m.setHMIAppId(pAppId, d.params.application.appID)
+        end)
+      m.getMobileSession(pAppId):ExpectResponse(corId, { success = true, resultCode = "SUCCESS" })
+      :Do(function()
+          m.getMobileSession(pAppId):ExpectNotification("OnHMIStatus",
+            { hmiLevel = "NONE", audioStreamingState = "NOT_AUDIBLE", systemContext = "MAIN" })
+        end)
+    end)
+end
+
+function m.unregisterApp(pAppId)
```

Comment:
Function is not used in scripts, please remove.

Reply:
@GetmanetsIrina Done in b0c29ef